pom.xml

```diff
@@ -5,8 +5,8 @@
 
     <properties>
         <java.version>17</java.version>
-        <hapi.fhir.jpa.server.starter.revision>3</hapi.fhir.jpa.server.starter.revision>
-        <clinical-reasoning.version>3.26.0</clinical-reasoning.version>
+        <hapi.fhir.jpa.server.starter.revision>2</hapi.fhir.jpa.server.starter.revision>
```

Comment:
jpa.server.starter.revision went down? Is that right? Shouldn't revision numbers go up?